Makefile.checker

```diff
@@ -34,6 +34,16 @@ CHECKMLLIBFILE := checker/.mllibfiles
 CHECKERDEPS := $(addsuffix .d, $(CHECKMLDFILE) $(CHECKMLLIBFILE))
 -include $(CHECKERDEPS)
 
+# Copied files
+checker/esubst.mli: kernel/esubst.mli
+	cp -a $< $@
```

Comment:
How about adding a simple "DO NOT EDIT" header to the file, instead of just copying it? That's easy to do and would address @SkySkimmer 's concerns.

Reply:
Ok, warning added, not that is very visible but yeah; another choice would be to actually rename the files.